src/fairseq2/models/wav2vec2/asr/model.py

```diff
@@ -161,8 +169,12 @@ def compute_loss(
             zero_infinity=True,
         )
 
+        # (S, N, T) -> (N, S, T)
+        lprobs = lprobs.transpose(0, 1)
+
+        return Wav2Vec2AsrLoss(loss, lprobs)
+
 
-@final
```

Comment:
Is this removed accidentally ?

Reply:
We have decided to remove @final decorator from metric bags for easier authoring. There are use cases where a training recipe might need to extend an existing metric bag with additional metrics (e.g. custom WER handling).